lib/features/support/app_automate_driver.rb

```diff
@@ -27,13 +28,27 @@ def initialize(username, access_key, local_id, target_device, app_location, loca
     @access_key = access_key
     @local_id = local_id
     app_url = upload_app(username, access_key, app_location)
+
+    # Sets up identifiers for ease of connecting jobs
+    project_id = ENV['BUILDKITE'] ? ENV['BUILDKITE_PIPELINE_NAME'] : "local"
+    build_id = ENV['BUILDKITE'] ? "#{ENV['BUILDKITE_BRANCH']} #{ENV['BUILDKITE_BUILD_NUMBER']}" : SecureRandom.uuid
+    build_name = "#{@device_type} #{ENV['BUILDKITE_STEP_KEY']} #{ENV['BUILDKITE_RETRY_COUNT']}"
+
+    $logger.warn "Appium driver initialised for:"
+    $logger.warn "    project : #{project_id}"
+    $logger.warn "    build   : #{build_id}"
+    $logger.warn "    name    : #{build_name}"
```

Comment:
Hmm, I was using it due to the fact our logger uses warn as it's base reporting level, however I could change that to notify and use that instead which would be more appropriate.  I didn't want to introduce too much new noise however.

Reply:
Ok, hadn't realised that warn was the base.    If it can configured to allow us to log at info without introducing noise from elsewhere then that would be great, but obviously not worth too much time investing in it.

Reply:
I think I'll address this in a quick PR setting the default to `INFO` but allowing a `quiet` option that sets it to `ERROR`

Reply:
PR here: #77